x-pack/plugins/observability_solution/apm/public/components/app/mobile/service_overview/geo_map/embedded_map.test.tsx

```diff
@@ -10,27 +10,15 @@ import { DataView } from '@kbn/data-views-plugin/common';
 import React from 'react';
 import { EmbeddedMap } from './embedded_map';
 import { KibanaContextProvider } from '@kbn/kibana-react-plugin/public';
-import { embeddablePluginMock } from '@kbn/embeddable-plugin/public/mocks';
 import { MockApmPluginContextWrapper } from '../../../../../context/apm_plugin/mock_apm_plugin_context';
 import { MemoryRouter } from 'react-router-dom';
 import { MapTypes } from '../../../../../../common/mobile/constants';
 
 describe('Embedded Map', () => {
   it('it renders', async () => {
-    const mockSetLayerList = jest.fn();
-    const mockUpdateInput = jest.fn();
-    const mockRender = jest.fn();
-    const mockReload = jest.fn();
-
-    const mockEmbeddable = embeddablePluginMock.createStartContract();
-    mockEmbeddable.getEmbeddableFactory = jest.fn().mockImplementation(() => ({
-      create: () => ({
-        setLayerList: mockSetLayerList,
-        updateInput: mockUpdateInput,
-        render: mockRender,
-        reload: mockReload,
-      }),
-    }));
+    const mockMapsApi = {
```

Comment:
How about renaming to `mockMapsStartService`. `MapApi` is the name of the map embeddable API.